cmd/utils.go

```diff
@@ -91,16 +94,21 @@ func ParseFlags(scenarioDetail *models.ScenarioDetail, args []string, scenarioCo
 				return nil, nil, err
 			}
 		}
-
-		if value != nil && field.Type != typing.File {
-			environment[*field.Variable] = *value
-		} else if value != nil && field.Type == typing.File {
-			fileSrcDst := strings.Split(*value, ":")
-			volumes[fileSrcDst[0]] = fileSrcDst[1]
+		if value != nil {
+			if field.Type != typing.File {
+				environment[*field.Variable] = *value
+			} else if field.Type == typing.File {
+				if field.MountPath != nil {
+					volumes[*field.MountPath] = *value
+					environment[*field.Variable] = *value
+				} else {
+					fileSrcDst := strings.Split(*value, ":")
```

Comment:
@paigerube14 I'm not so convinced to keep the double usage (with or without `:`). I understand that might be helpful in some conditions to have control on the destination mount path dynamically, but I've the feeling that this could lead to a security issue allowning the user to override files in the container. For that reason I was thinking to set the mount path as mandatory for file type. I made a temporary fix on this branch in the validator to make the tests pass, but if we do agree on that point, instead of assigning the local path to the mountPath I would rise an error. Let's discuss about that.